pkg/comp-functions/functions/vshnpostgres/pgqexporter_config.go

```diff
@@ -0,0 +1,55 @@
+package vshnpostgres
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+
+	"github.com/crossplane/function-sdk-go/proto/v1beta1"
+	xkube "github.com/vshn/appcat/v4/apis/kubernetes/v1alpha2"
+	vshnv1 "github.com/vshn/appcat/v4/apis/vshn/v1"
+	"github.com/vshn/appcat/v4/pkg/comp-functions/runtime"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+/*
+	This code ensures we have minimalistic postgresql-exporter configuration
+	It prevents high memory usage for big databases
+*/
+//go:embed scripts/queries.yml
+var queries string
+
+func PgExporterConfig(ctx context.Context, comp *vshnv1.VSHNPostgreSQL, svc *runtime.ServiceRuntime) *v1beta1.Result {
+	err := svc.GetObservedComposite(comp)
```

Comment:
Do you disable somewhere the current exporter?

Reply:
We don't disable anything, we just override the default configs from stackgres.